docs/source/components/nodes/stereo_depth.rst

```diff
@@ -375,8 +375,26 @@ or roughly 35cm.
 
 See `these examples <https://github.com/luxonis/depthai-experiments/tree/master/gen2-camera-demo#real-time-depth-from-depthai-stereo-pair>`__ for how to enable Extended Disparity.
 
+Disparity shift to lower min depth perception
+---------------------------------------------
+
+Another option to perceive closer depth range is to use disparity shift. Disparity shift will shift the starting point
+of the disparity search, which will significantly decrease max depth perception, but it will also decrease min depth perception.
+
+.. image:: https://user-images.githubusercontent.com/18037362/189375017-2fa137d2-ad6b-46de-8899-6304bbc6c9d7.png
+
+**Left graph** shows min and max disparity and depth for OAK-D (7.5cm baseline, 800P resolution, ~70° HFOV) by default (disparity shift=0). Since hardware (stereo
```

Comment:
Link to how depth is calculated from disparity. Preferably using intrinsic matrix data to retrieve focal length so it works for all lenses.